### ISSUE TITLE
P1 Angled Panel editor models use incorrect piston texture

### ISSUE THREAD

**Issue by Luke18033** [BEEmod/BEE2-items#3310]
The Angled Panel piston should be using the same reflective texture as the piston platform, as it does in-game. It currently uses an incorrect metal texture.
![p1panels](https://user-images.githubusercontent.com/19228201/82377111-08eda880-99d8-11ea-857e-b2c316406414.png)
(and yes I just reused Lautaro's image with the text changed, I can't take my own right now)

**Comment by Luke18033** [comment]
Also, we should probably make a specific cubemap texture for editor models, which shows a room using editor textures instead of in-game ones. Or maybe we should just get rid of the reflections in the editor models entirely?

**Comment by TeamSpen210** [comment]
It needs the reflections to look like a piston, almost all of the appearance is from the cubemap.

**Comment by Luke18033** [comment]
Yeah but maybe for the editor model we change the reflections to be more diffuse, like using phong effects or something. I just don't feel like a highly reflective metal surface really fits the puzzlemaker aesthetic.
